opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/TracerAutoConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package io.opentracing.contrib.spring.web.autoconfig;
+
+import java.util.logging.Logger;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.opentracing.NoopTracer;
+import io.opentracing.NoopTracerFactory;
+import io.opentracing.Tracer;
+
+/**
+ * @author Pavol Loffay
+ */
+@Configuration
+public class TracerAutoConfiguration {
+    private final static Logger log = Logger.getLogger(TracerAutoConfiguration.class.getName());
+
+    @Bean
+    @ConditionalOnMissingBean(Tracer.class)
+    public Tracer noopTracer() {
+        log.severe("Tracer bean is not configured! Switching to " + NoopTracer.class.getName());
```

Comment:
Quoting the javadoc for [`SEVERE`](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html#SEVERE)

> SEVERE is a message level indicating a serious failure.
> 
> In general SEVERE messages should describe events that are of considerable importance and which will prevent normal program execution. They should be reasonably intelligible to end users and to system administrators. This level is initialized to 1000.

Remember that Tracing is *not* a business code and might be a "nice thing to have", rather than "something I absolutely need". So, I'd rather have it in a lower log level and let users that really care about Tracing to do extra checks, if they so wish.